integrations/prodigy_ner_fashion/project.yml

```
@@ -0,0 +1,126 @@
+
+# Assets that should be downloaded or available in the directory. We're shipping
+# them with the project, so they won't have to be downloaded. But the
+# 'project assets' command still lets you verify that the checksums match.
+assets:
+  - dest: "assets/${vars.train}.jsonl"
+    checksum: "63373dd656daa1fd3043ce166a59474c"
+    description: "JSONL-formatted training data exported from Prodigy, annotated with `FASHION_BRAND` entities (1235 examples)"
+  - dest: "assets/${vars.dev}.jsonl"
+    checksum: "5113dc04e03f079525edd8df3f4f39e3"
+    description: "JSONL-formatted development data exported from Prodigy, annotated with `FASHION_BRAND` entities (500 examples)"
+  # Patterns are not used for training but we distribute them for reference
+  - dest: "assets/${vars.patterns}.jsonl"
+    checksum: "4070316032ce36a01b7d1e8ecb387a8b"
+    description: "Patterns file generated with `sense2vec.teach` and used to pre-highlight during annotation (100 patterns)"
+
+# Workflows are sequences of commands (see below) executed in order. You can
+# run them via "spacy project run [workflow]". If a commands's inputs/outputs
+# haven't changed, it won't be re-run.
+workflows:
+  all:
+    - db-in
+    - data-to-spacy
+    - train_spacy
+    - evaluate
+
+  all_prodigy:
+    - db-in
+    - data-to-spacy
+    - train_prodigy
+    - evaluate
+
+# Project commands, specified in a style similar to CI config files (e.g. Azure
+# pipelines). The name is the command name that lets you trigger the command
+# via "spacy project run [command] [path]". The help message is optional and
+# shown when executing "spacy project run [optional command] [path] --help".
+commands:
+  - name: "db-in"
+    help: "Load data into prodigy"
+    script:
+      - "python -m prodigy db-in ner_${vars.train} assets/${vars.train}.jsonl"
+      - "python -m prodigy db-in ner_${vars.dev} assets/${vars.dev}.jsonl"
+    deps:
+      - "assets/${vars.train}.jsonl"
+      - "assets/${vars.dev}.jsonl"
+
+  - name: "data-to-spacy"
+    help: "Convert data from prodigy to spaCy's binary format"
+    script:
+      - "python -m prodigy data-to-spacy corpus/ --ner ner_${vars.train},ner_${vars.dev} --eval-split ${vars.eval_split}"
+
+    outputs:
+      - "corpus/${vars.train}.spacy"
+      - "corpus/${vars.dev}.spacy"
+
+  - name: "train_spacy"
+    help: "Train a named entity recognition model with spaCy"
+    script:
+      - "python -m spacy train configs/${vars.config} --output training/ --paths.train corpus/train.spacy --paths.dev corpus/dev.spacy --gpu-id ${vars.gpu_id}"
+    deps:
+      - "corpus/train.spacy"
+      - "corpus/dev.spacy"
+    outputs:
+      - "training/model-best"
+
+  - name: "train_prodigy"
+    help: "Train a named entity recognition model with prodigy"
+    script:
+      - "python -m prodigy train training/ -c configs/config.cfg --ner ner_${vars.train},ner_${vars.dev} --eval-split ${vars.eval_split} --gpu-id ${vars.gpu_id}"
```

The `ner_${vars.train}` etc. here is pretty confusing, since this is just an arbitrary dataset name. I think you should move this into separate variables `train_dataset` and `eval_dataset` at the top. You can then use the same variable names as the file names for the exported `.spacy` files.

Also, instead of using the `--eval-split` here, we should use the evaluation set directly, e.g.:

```suggestion
      - "python -m prodigy train training/ -c configs/config.cfg --ner ${vars.train_dataset},eval:${vars.eval_dataset} --gpu-id ${vars.gpu_id}"
```